src/main/java/uk/gov/ons/census/casesvc/service/UacProcessor.java

```diff
@@ -57,24 +57,27 @@ public UacQidLink saveUacQidLink(Case caze, int questionnaireType) {
     return uacQidLink;
   }
 
-  public void logEvent(UacQidLink uacQidLink, String eventDescription) {
+  public void logEvent(
+      UacQidLink uacQidLink, String eventDescription, LocalDateTime eventMetaDataDateTime) {
```

Comment:
suggestion: overload this method so that this method gets called with `eventMetaDataDateTime` as null if the parameter is not specified